docs/index.md

```diff
@@ -0,0 +1,143 @@
+# Usage
+All Commands:
+
+[embedmd]:# (_intermediate/help.txt)
+```txt
+A command-line application to lint Grafana dashboards.
+
+Usage:
+  dashboard-linter [flags]
+  dashboard-linter [command]
+
+Available Commands:
+  completion  Generate the autocompletion script for the specified shell
+  help        Help about any command
+  lint        Lint a dashboard
+  rules       Print documentation about each lint rule.
+
+Flags:
+  -h, --help   help for dashboard-linter
+
+Use "dashboard-linter [command] --help" for more information about a command.
+```
+
+## Completion
+[embedmd]:# (_intermediate/completion.txt)
+```txt
+Generate the autocompletion script for dashboard-linter for the specified shell.
+See each sub-command's help for details on how to use the generated script.
+
+Usage:
+  dashboard-linter completion [command]
+
+Available Commands:
+  bash        Generate the autocompletion script for bash
+  fish        Generate the autocompletion script for fish
+  powershell  Generate the autocompletion script for powershell
+  zsh         Generate the autocompletion script for zsh
+
+Flags:
+  -h, --help   help for completion
+
+Use "dashboard-linter completion [command] --help" for more information about a command.
+```
+
+## Lint
+[embedmd]:# (_intermediate/lint.txt)
+```txt
+Returns warnings or errors for dashboard which do not adhere to accepted standards
+
+Usage:
+  dashboard-linter lint [dashboard.json] [flags]
+
+Flags:
+  -h, --help      help for lint
+      --strict    fail upon linting error or warning
+      --verbose   show more information about linting
```

Comment:
Does verbose show the readme definitions for the rules added here or do we have to refer to github for that?

Reply:
No, verbose just shows *every* result, including success. By default, success messages are suppressed. This option is like the opposite of `quiet` which does not exist as an option, since it's the default behavior.